config/locales/views/annotations/en.yml

```diff
@@ -2,10 +2,12 @@ en:
   annotations:
     count: 'Number of times used: '
     default_deduction: 'This annotation will have no deduction. Admins can configure deductions in assignment settings.'
+    empty_uncategorized: 'Currently, no annotations without a category exist.'
```

Comment:
You can remove the word "currently". Also use the phrase "one-time only" (see comment below).